### ISSUE TITLE
Change Login Screen Intro Video to "What is Fediverse"

### ISSUE THREAD

**Issue by jeffsikes** [feditext/feditext#116]
**Is your feature request related to a problem? Please describe.**
Now that Feditext is expanding it's reach beyond Mastodon, I'm wondering if it would make sense to change the video that currently appears during the login process (currently "[What is Mastodon]"(https://www.youtube.com/watch?v=IPSbNdBmWKE) ) to a more universal "[What is the fediverse](https://framatube.org/w/4294a720-f263-4ea4-9392-cf9cea4d5277)" version.

**Describe the solution you'd like**
If you're not planning on altering the login process in a broader sense, consider making this change to increase awareness of the fediverse outside of Mastodon.

**Describe alternatives you've considered**
Rather than a general video, adding the logo and basic information of the instance (from nodeinfo or fedidb). I've seen this in a few other apps recently.

Another alternative - it's probably a bit much, but since there is kind of a feature matrix in regards to what is supported in the app per platform, add a short statement of what may be limited for the entered platform/version would be nice, or at least after logging in so you know what to expect.

**Additional context**
Nothing else to add. Thanks for your consideration, hope this was a useful contribution. Enjoying the app, thank you.

**Comment by VyrCossont** [comment]
Agreed that this is due for a rework. We're already fetching NodeInfo and the v1 instance API, and should be able to display the blurb, links to the relevant server software's home page, and which major features are enabled.
